src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -296,32 +301,121 @@ public Entity deserializeEntity(byte[] data, World world, boolean preserveUUID)
 		throw new UnimplementedOperationException();
 	}
 
+	/**
+	 * Gets the translation key for a {@link Material} that {@link Material#isBlock() is a block}.
+	 * @param material the material to translate
+	 * @return a string of the structure "block.{@literal <namespace>}.{@literal <material>}" (e.g. "block.minecraft.stone"), or null if not a block.
+	 */
 	@Override
 	public String getBlockTranslationKey(Material material)
```

Comment:
Even though this is not specified in the API, could you add the Nullable anotation for the output, and NotNull annotation for the input?

Eh, this includes all of the other cases as well, where necessary.

Reply:
What annotations do you want? Jetbrains or javax?

Reply:
I think we use jetbrains